src/5.0_to_5.1/__test_resources__/smallContentServer.ts

```diff
@@ -0,0 +1,1240 @@
+                                canRead: true,
+                                canWrite: true,
+                              },
+                            },
+                          },
+                        },
+                        "478": {
+                          entry: {
+                            isDirectory: true,
+                            owners: ["admin"],
+                            readers: ["admin"],
+                            timestamp: 1669821833579,
+                            lastEditor: "admin",
+                            canRead: true,
+                            canWrite: true,
+                          },
+                          children: {
+                            "478_metadata": {
+                              entry: {
+                                content: '{"name":"Dashboard test  (copy)"}',
+                                isDirectory: false,
+                                owners: ["admin"],
+                                readers: ["admin"],
+                                timestamp: 1669821833579,
+                                lastEditor: "admin",
+                                canRead: true,
+                                canWrite: true,
+                              },
+                            },
+                          },
+                        },
+                        "31b_metadata": {
+                          entry: {
+                            content: '{"isFolder":true,"name":"lil mad"}',
+                            isDirectory: false,
+                            owners: ["admin"],
+                            readers: ["admin"],
+                            timestamp: 1669817887286,
+                            lastEditor: "admin",
+                            canRead: true,
+                            canWrite: true,
+                          },
+                        },
+                        c9c: {
+                          entry: {
+                            isDirectory: true,
+                            owners: ["admin"],
+                            readers: ["admin"],
+                            timestamp: 1669817918087,
+                            lastEditor: "admin",
+                            canRead: true,
+                            canWrite: true,
+                          },
+                          children: {
+                            c9c_metadata: {
+                              entry: {
+                                content: '{"name":"sub dash"}',
+                                isDirectory: false,
+                                owners: ["admin"],
+                                readers: ["admin"],
+                                timestamp: 1669817918087,
+                                lastEditor: "admin",
+                                canRead: true,
+                                canWrite: true,
+                              },
+                            },
+                          },
```

Comment:
It seems to me that `c9c` is in `structure` but neither in `content` nor `thumbnails`.
If so, can you remove it ?

Reply:
Yes, good catch, my bad!